### ISSUE TITLE
Focus styles are not visible in pagination component

### ISSUE THREAD

**Issue by jgiardino** [bf2fc6cc711aee1a0c2a/kas-ui#158]
One of the WCAG requirements is to [ensure focus is visible](https://www.w3.org/WAI/WCAG21/quickref#focus-visible) when an element has focus, to enable users who navigate with a keyboard to see where they are in the UI.

In Chrome, when I tab through the [PF pagination](https://www.patternfly.org/v4/components/pagination) component, I see a blue border ring indicating which element has focus:
<img width="237" alt="image" src="https://user-images.githubusercontent.com/21063328/104211540-2f969e80-5402-11eb-880c-e0e794e7f5c9.png">

In our current implementation, I'm not seeing focus rings display. For this issue, we should identify why this is the case and address it.
